### PR TITLE
XRT-1193: Added lua scripting support for all datatypes and arrays of datatypes

### PR DESCRIPTION
**Pull request by dpazj**
* Moved example lua scripts to xrt-examples

### CODE REVIEW

File: scripts/Dockerfile.ble-sim [hunk 1]
```diff
@@ -27,6 +27,10 @@ WORKDIR /ble-sim/
 RUN scripts/download-deps.sh
 RUN scripts/build.sh
 
+WORKDIR /
+#RUN git clone --branch v1.2-branch-sdk-update https://github.com/IOTechSystems/xrt-examples
+RUN git clone --branch XRT-1193-branch https://github.com/IOTechSystems/xrt-examples
```
Comment (Tylerg2471) [inline scripts/Dockerfile.ble-sim]: Should the branch be changed before merge?

**Review by Tylerg2471 — CHANGES_REQUESTED** [review]
One minor change to look at

File: scripts/Dockerfile.ble-sim (hunk 1 above)
Comment (dpazj) [reply scripts/Dockerfile.ble-sim]: Yeah sorry, didn't mean to leave that in: if everything else is good then I will fix that.

**Review by Tylerg2471 — APPROVED** [review]
LGTM

**Pull request merged**